### ISSUE TITLE
Null Safety Migration

### ISSUE THREAD

**Issue by jakesterrr123** [Baseflow/flutter_cached_network_image#532]
## 🏗 Enhancement Proposal
Any plans to migrate the package to null safety?

**Comment by georgeherby** [comment]
Duplicate of #503

**Comment by renefloor** [comment]
Plans for sure. I just need 2 things:
1) http needs to have a compatible library, apparently that is now unblocked to be migrated, see: https://twitter.com/filiphracek/status/1345554569573482497
2) time to migrate or a PR by somebody that migrates the library.

Closing this issue as duplicate.
